doc/user/content/sql/create-table.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "CREATE TABLE"
+description: "`CREATE TABLE` creates a non-streaming, in-memory data source."
+menu:
+  main:
+    parent: 'sql'
+---
+
+`CREATE TABLE` creates a non-streaming, in-memory data source.
+
+## Conceptual framework
+
+Tables store non-streaming data that is inserted via [INSERT](../insert) statements.
+
+### When to create a table
+
+You might want to create a table when...
+
+- You want to manually insert rows of data into Materialize.
+- You want to test Materialize's features without setting up a data stream.
+
+## Syntax
+
+{{< diagram "create-table.svg" >}}
+
+Field | Use
+------|-----
+_table&lowbar;name_ | A name for the table.
+_col&lowbar;name_ | The name of the column to be created in the table.
+_col&lowbar;type_ | The data type of the column indicated by _col_name_.
+_NOT NULL_ | Provide a `NOT NULL` constraint if a column is not nullable. All columns without this constraint will be nullable.
```

Comment:
nit: s/`_NOT NULL_`/`**NOT NULL**`/

I'm a little confused by the wording on this constraint; creating a table ostensibly gives you control over a table's behavior, but saying "if a column is not nullable" makes it sound like you're trying to match or inherit this constraint from another source. If it's the case that you are controlling the behavior, could I get you to reword it slightly? Feel free to work from this, if so:

> `**NOT NULL** | Do not allow the column to contain _NULL_ values. Columns without this constraint can contain _NULL_ values`.